.github/workflows/build.yml

```
@@ -0,0 +1,40 @@
+name: Validate and Build
```

I'd prefer something cleaner (one word) because currently, the messages aren't as clear as they could be:
```
Validate and Build / validate (pull_request) Successful in 14s
Validate and Build / build (pull_request) Successful in 25s
```

It would be more understandable if it was something like:
```
Compile / validate (pull_request) Successful in 14s
Compile / build (pull_request) Successful in 25s
```

Note, this is not final, I'm just proposing something shorter (build, compile, that sort of names).
```suggestion
name: Compile
```